src/MembershipsSDKAdapter.js

```diff
@@ -0,0 +1,109 @@
+import {BehaviorSubject, throwError} from 'rxjs';
+import {DestinationType, MembershipsAdapter} from '@webex/component-adapter-interfaces';
+
+/**
+ * Gets the active members in a meeting
+ *
+ * @private
+ * @param {object} members Members object from meeting, keyed by ID
+ * @returns {Array}
+ * @memberof MembershipsSDKAdapter
+ */
+function getActiveMembers(members) {
+  return Object.values(members)
+    .filter((member) => !member.isInMeeting)
+    .map((member) => ({
+      id: member.id,
+    }));
+}
+
+export default class MembershipsSDKAdapter extends MembershipsAdapter {
+  constructor(datasource) {
+    super(datasource);
+
+    this.membershipSubjects = {};
+  }
+
+  /**
+   * Returns an observable subject for the given membership.
+   *
+   * @private
+   * @param {string} membershipID  ID of the destination for which to get members
+   * @param {DestinationType} destinationType  type of the membership destination
+   * @returns {BehaviorSubject}
+   */
+  getOrCreateMembershipSubject(membershipID) {
+    if (!this.membershipSubjects[membershipID]) {
+      // Behavior subject will keep the last emitted object for new subscribers
+      // https://rxjs.dev/guide/subject#behaviorsubject
+      this.membershipSubjects[membershipID] = new BehaviorSubject([]);
+    }
+
+    return this.membershipSubjects[membershipID];
+  }
+
+  /**
+   * Returns an observable that emits a Membership object.
+   * Whenever there is an update to the membership, the observable
+   * will emit a new updated Membership object, if datasource permits.
+   *
+   * @param {string} destinationID  ID of the destination for which to get members
+   * @param {DestinationType} destinationType  type of the membership destination
+   * @returns {external:Observable.<Membership>} Observable stream that emits membership data
+   * @memberof MembershipAdapter
+   */
+  getMembersFromDestination(destinationID, destinationType) {
+    if (destinationType !== DestinationType.MEETING) {
+      return throwError(new Error(`getMembersFromDestination for ${destinationType} is not currently supported.`));
+    }
+
+    const meeting = this.datasource.meetings.getMeetingByType('id', destinationID);
+
+    if (!meeting) {
+      return throwError(new Error(`Meeting ${destinationID} not found.`));
+    }
+
+    const membershipID = `${destinationType}-${destinationID}`;
+
+    if (!this.membershipSubjects[membershipID]) {
+      // Behavior subject will keep the last emitted object for new subscribers
+      // https://rxjs.dev/guide/subject#behaviorsubject
+      this.membershipSubjects[membershipID] = new BehaviorSubject({
```

Comment:
You should initialize with the data after you fetched it. No need to emit twice to send empty + initial data